test/run_cli_test.py

```diff
@@ -0,0 +1,27 @@
+import pathlib
+import pytest
+
+from click.testing import CliRunner
+
+import run
+from libs import build_params
+
+
+@pytest.mark.parametrize('dataset_name', ['JHU', 'CDS'])
+def test_runs_all_interventions_on_one_state(tmpdir, dataset_name):
+    runner = CliRunner()
+
+    runner.invoke(run.main, [
+        'run-model', dataset_name, '--state', 'MA', '--output-dir', tmpdir
+    ], catch_exceptions=False)
```

Comment:
FWIW I think this test is more likely to slow us down (as we add additional models, rework how datasets work, etc.) than to catch a regression.

In other words, I expect the "spec" for run.py to be rapidly evolving.  I'm not sure it's the best place to be testing right now.

Reply:
Yeah I go back and forth - this test feels high level enough to me to serve as a stop gap - I think we should make sure that no changes break the basic command line run function.

I think the test is easy enough to update that there shouldn't be a ton of maintenance? happy to figure out another way to go about it though